.travis.yml

```diff
@@ -12,6 +12,13 @@ jobs:
       before_script: ./scripts/decrypt-app-credentials.sh
       script: ./scripts/test-hosting.sh
       after_script: skip
+  include:
```

Comment:
I don't see the "hosting functional test" running on this PR .... I wonder if your `include` is wiping out the previous value and instead you just want to add another `-stage` to the hosting test.